src/tool/subcommands/config_cmd.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::io::Write;
+
+use crate::cli_shared::cli::CliOpts;
+use anyhow::Context;
+use clap::Subcommand;
+
+#[derive(Debug, Subcommand)]
+pub enum ConfigCommands {
+    /// Dump current configuration to standard output
+    Dump {
+        /// Forest daemon flags to override current configuration
+        #[command(flatten)]
+        opts: CliOpts,
```

Comment:
I don't think `CliOpts` are appropriate here. For example, how does `forest-tool config dump --exit-after-init` make sense?